### PR TITLE
Updated keymgr_dpe key derivation test

### PR DESCRIPTION
**Pull request by rsahita**
Updated keymgr_dpe key derivation test

### CODE REVIEW

File: sw/device/tests/keymgr_dpe_key_derivation_test.c [hunk 1]
```diff
@@ -74,5 +74,32 @@ bool test_main(void) {
...
+  genparams.slot_src_sel = 1;
+
+  dif_keymgr_dpe_output_t key;
+
+  CHECK_STATUS_OK(keymgr_dpe_testutils_generate(&keymgr_dpe, &genparams, &key));
+  LOG_INFO("KeymgrDpe generated key:");
+  for (uint32_t i=0; i<2; i++) {
+	  for (uint32_t j=0; j<8; j++) {
+		  LOG_INFO("%x ", key.value[i][j]);
+	  }
```
Comment (rsahita) [inline sw/device/tests/keymgr_dpe_key_derivation_test.c:101]:
```suggestion
      for (uint32_t j=0; j<8; j++) {
          LOG_INFO("%x ", key.value[i][j]);
      }
```

File: sw/device/tests/keymgr_dpe_key_derivation_test.c [hunk 2]
```diff
@@ -74,5 +74,32 @@ bool test_main(void) {
...
+
+  dif_keymgr_dpe_output_t key;
+
+  CHECK_STATUS_OK(keymgr_dpe_testutils_generate(&keymgr_dpe, &genparams, &key));
+  LOG_INFO("KeymgrDpe generated key:");
+  for (uint32_t i = 0; i < 2; i++) {
+    for (uint32_t j = 0; j < 8; j++) {
+      LOG_INFO("%x ", key.value[i][j]);
+    }
+  }
```
Comment (engdoreis) [inline sw/device/tests/keymgr_dpe_key_derivation_test.c:102]:
```suggestion
  for (size_t i = 0; i < ARRAYSIZE(key.value); ++i) {
    for (size_t j = 0; j < ARRAYSIZE(key.value[0]); ++j {
      LOG_INFO("%x ", key.value[i][j]);
    }
  }
```

**Review by engdoreis — APPROVED** [review]
Thanks for the PR, I left a code style comment below. Other than that LGTM.

**Comment by rsahita** [comment]
@moidx - need some help to see why CI failed the last time (it had succeeded before) - can you PTAL and merge?

**Comment by moidx** [comment]
Closing/re-opening PR to trigger CI run.

**Pull request merged**